tests/benchmarks/bench_dg_amr.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "bench.hpp"
+
+#ifdef MFEM_USE_BENCHMARK
+
+/*
+  This benchmark is inspired by the CEED's bake-off problems to benchmark the
+  performance of the action of the partial assembly (PA) of DG Convection on
+  nonconforming meshes.
+
+   * --benchmark_filter=BK_DG/[1-max_order]/[prob]
```

Comment:
Shouldn't there be the size of the problem between the kernel name and the order?

Reply:
done in [d763963](https://github.com/mfem/mfem/pull/3105/commits/d763963c2406d2647210bbe5696f566438f12f43).